### PR TITLE
fix(pci.db): update private network when changing region

### PR DESCRIPTION
**Pull request by abullet33**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #datatr-77 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
update private network when changing region

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js [hunk 1]
```diff
@@ -189,7 +189,10 @@ export default class {
   }
 
   updateRegion(region) {
-    this.model.region = region;
+    if (this.model.region !== region) {
```
Comment (karim-ovh) [inline packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js:192]: Why you don't compare the region name like the following one.

Comment (karim-ovh) [inline packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js:192]: `if (this.model.region.name !== region.name) {`

**Review by jperchoc — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js (hunk 1 above)
Comment (abullet33) [reply packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js]: Hello, yes that's true! Fixed :)

Comment (abullet33) [reply packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js]: Fixed

**Review by jperchoc — APPROVED** [review]

**Review by karim-ovh — APPROVED** [review]

**Review by JacquesLarique — APPROVED** [review]

**Pull request closed without merging**